whydoge.html

```diff
@@ -0,0 +1,190 @@
+</head>
+<body id="page-top">
+    <!-- Navigation-->
+    <nav class="navbar navbar-expand-lg bg-secondary text-uppercase fixed-top" id="mainNav">
+        <div class="container-fluid">
+            <a class="navbar-brand js-scroll-trigger" href="#page-top"><img src="assets/img/dogecoin-300.png">&nbsp;Dogecoin</a>
+            <button class="navbar-toggler navbar-toggler-right text-uppercase font-weight-bold bg-primary text-secondary rounded" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                Menu
+                <i class="fas fa-bars"></i>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="index.html" data-i18n="dgc-back-to-main">Back to Main Page</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <!-- Masthead-->
+    <header class="masthead bg-primary text-white text-center" style="position:relative;">
+        <div id="particles" style="position:absolute;top:0;bottom:0;left:0;right:0;"></div>
+        <div class="container d-flex align-items-center flex-column">
+            <div class="row">
+                <div class="col-md-12">
+                    <img class="masthead-avatar mb-1" style="margin-top:-3rem;" src="assets/img/doge.png" alt="" />
+                    <h1 class="font-weight-light text-secondary mb-2">
+                        <span>Support Dogecoin</span>
+                    <h1>
+                    <!-- Masthead Subheading-->
+                    <p class="masthead-subheading font-weight-light text-secondary mb-0 mt-4"> We are a fast-growing community of shibas. There are many ways to contribute if you are interested.
+                    </p>
+                 </div>
+             </div>
+         </div>
+     </div>
+ </header>
+ <!-- Portfolio Section-->
+ <section class="page-section wallet-guide-detail" id="windows">
+    <div class="container">
+        <!-- Portfolio Grid Items-->
+        <div class="row text-center justify-content-center">
+            <div class="col-lg-4 col-md-6 mb-4">
+                <div class="support-content">
+                    <i class="fa fa-bullhorn text-secondary mb-2"></i>
+                    <h3 data-i18n="dgc-support-spread-the-word" class="heading mb-4">
+                        Spread the Word
+                    </h3>
+                    <div class="text">
+                        Become a self-appointed Dogecoin ambassador. Tell your friends and family about it. Join <a class="btn btn-sm btn-secondary" href="https://www.reddit.com/r/dogecoin/" target="_blank">r/dogecoin</a>. Ask your favorite shops to let you pay with Dogecoin. You can even donate Dogecoin to someone in need. Much noble. So respect. 
```

Comment:
Reframe the `asking shops.... to accept dogecoins` sentence to something that means like you can also pay with dogecoins at your fav shops , find the list of stores here https://dogecoins.com/spend/
We can't just ask random shops to let them accept doge reframing it will make it a bit clear